tox.ini

```diff
@@ -126,6 +126,11 @@ envlist =
     benchmarks-{py27,py34,py35,py36,py37}
 
 [testenv]
+# Always re-run `setup.py develop` to ensure the proper C-extension .so files are created
+# DEV: If we don't do this sometimes CircleCI gets messed up and only has the py27 .so
+#      meaning running on py3.x will fail
+#      https://stackoverflow.com/questions/57459123/why-do-i-need-to-run-tox-twice-to-test-a-python-package-with-c-extension
+commands_pre={envpython} {toxinidir}/setup.py develop
```

Comment:
not sure i understand that but ok

Reply:
honestly... same... this was super duper confusing to debug, this was the only thing that fixed it.